src/controllers/itemDetails/index.js

```diff
@@ -167,7 +167,30 @@ function renderTrackSelections(page, instance, item, forceReload) {
         return;
     }
 
-    const mediaSources = item.MediaSources;
+    var mediaSources = item.MediaSources;
+
+    var resolutionNames = [];
+    var sourceNames = [];
+    mediaSources.forEach(function (v) {
+        ((v.Name.endsWith('p') || v.Name.endsWith('i'))) && !Number.isNaN(parseInt(v.Name, 10) ? resolutionNames.push(v) : sourceNames.push(v);
```

Comment:
```suggestion
        (v.Name.endsWith('p') || v.Name.endsWith('i')) && !Number.isNaN(parseInt(v.Name, 10)) ? resolutionNames.push(v) : sourceNames.push(v);
```
The lint task was failing because there was a missing parenthesis.